### ISSUE TITLE
[0.5.35][docker image]livepeer binary path in entrypoint doesn't match real path

### ISSUE THREAD

**Issue by joegraviton** [livepeer/go-livepeer#2688]
**Describe the bug**

For docker image `livepeer/go-livepeer:0.5.35`, Entrypoint path is `/usr/bin/livepeer`.
But actual path is `/usr/local/bin/livepeer`.

Container will fail to start,  or even fail to show help:
```
docker run -it livepeer/go-livepeer:0.5.35 -help

docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/usr/bin/livepeer": stat /usr/bin/livepeer: no such file or directory: unknown.
ERRO[0000] error waiting for container: context canceled 

# working for 0.5.34, this will show help:
docker run -it livepeer/go-livepeer:0.5.34 -help
```

**To Reproduce**
1. Inspect image to check entrypoint path:
```
docker inspect livepeer/go-livepeer:0.5.35 | grep /usr/bin/livepeer -C 1
            "Entrypoint": [
                "/usr/bin/livepeer"
            ],

```
2. Start container from image and check actual livepeer path:

```
docker run -it --entrypoint /bin/bash livepeer/go-livepeer:0.5.35 which livepeer
/usr/local/bin/livepeer
```

**Expected behavior**

Path should match.

**Additional context**

The binary path changed to `/usr/local/bin/livepeer` since 0.5.35. In 0.5.34, it was correct.
Any one using this image in docker or Kubernetes will run into this error.

**Comment by leszko** [comment]
We had a similar issue here: https://github.com/livepeer/go-livepeer/issues/2633

@hjpotter92 could you look into this one?

**Comment by hjpotter92** [comment]
> **Describe the bug**
> 
> For docker image `livepeer/go-livepeer:0.5.35`, Entrypoint path is `/usr/bin/livepeer`. But actual path is `/usr/local/bin/livepeer`.
> 
> Container will fail to start, or even fail to show help:
> 
> ```
> docker run -it livepeer/go-livepeer:0.5.35 -help
> 
> docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/usr/bin/livepeer": stat /usr/bin/livepeer: no such file or directory: unknown.
> ERRO[0000] error waiting for container: context canceled 

Hey @joegraviton ; sorry that you're facing this issue. The entrypoint was indeed updated in the core repository as a fix for #2633 but there hasn't been any patch/minor release made to the repository, making the last tagged release to have a broken `ENTRYPOINT` currently.

I'd recommend keeping an eye out for the next release on the project; or try using `livepeer/go-livepeer:pr-2635` image (pr #2635 was where the bug was fixed). you can also try the `livepeer/go-livepeer:latest` docker image; which might contain some untested features (and could be unstable until a release is done)

**Comment by joegraviton** [comment]
@hjpotter92  no problem, I will close this issue.
